.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy Github Pages
+
+on:
+  push:
+    branches:
+      - mkdocs
```

Comment:
Is this intentional or, eg: a holdover from testing? `main` and `mkdocs` seem like they may eventually get out of sync if you're deploying from this branch. Totally up to you though!